docs/firebase.md

```diff
@@ -81,3 +81,5 @@ and filed under `origin/{accountId}` (the name of the service account) of the DA
 The test [Web Application](https://daq-project.firebaseapp.com/) (again, will have to substitute the appropriate `projectId`),
 should show a list of all accounts with ingested data. If nothing is showing here, or the `accountId` is missing,
 check the web dev console to see if there's any obvious errors.
+
```

Comment:
woo. And just to practice -- maybe take this one out (revert this change), but keep the other one!

Reply:
OK